### PR TITLE
update some npm dependencies because some had critical vulnerabilities

### PR DESCRIPTION
**Pull request by antonilol**

### CODE REVIEW

**Comment by kevkevinpal** [comment]
seems like the testing env didn't like it when we ran
`npm install && npm run build && docker build -t sphinxlightning/sphinx-relay . `

**Comment by antonilol** [comment]
weird, this is the same error i got in #377 (npm run build)

**Comment by antonilol** [comment]
i updated the vulnerable dependencies by hand and npm run build succeeded on my pc

**Comment by antonilol** [comment]
all successful except 2 times this random error `ERROR: for bob-lnd  Cannot start service bob-lnd: driver failed programming external connectivity on endpoint bob-lnd.sphinx (3366ecaf50b61f24f83dca791835eba0d6aecd7c79183940aa4d9b2ddd52acbc): Error starting userland proxy: listen tcp4 0.0.0.0:38882: bind: address already in use`

**Comment by kevkevinpal** [comment]
> all successful except 2 times this random error `ERROR: for bob-lnd Cannot start service bob-lnd: driver failed programming external connectivity on endpoint bob-lnd.sphinx (3366ecaf50b61f24f83dca791835eba0d6aecd7c79183940aa4d9b2ddd52acbc): Error starting userland proxy: listen tcp4 0.0.0.0:38882: bind: address already in use`

yea this is an issue on github side I can just rerun

File: package.json [hunk 1]
```diff
@@ -84,7 +91,7 @@
     "rncryptor-js": "0.0.1",
     "secp256k1": "3.8.0",
     "secure-random": "^1.1.2",
-    "sequelize": "5.19.3",
+    "sequelize": "^5.19.3",
```
Comment (kevkevinpal) [inline package.json:94]: is this just an update for the minor version? I think the `^` just updates the minor version number automatically

Comment (antonilol) [reply package.json:94]: yes, npm showed me it (or a dependency of it) had a vulnerability, so i ran `npm i sequelize`, npm did this automatically. npm adds that by default, i can remove it if you want

**Comment by antonilol** [comment]
> yea this is an issue on github side I can just rerun

i have seen this multiple times, and it was always bob-lnd that failed when it did, seems like some internal github thing uses that same port (38882)

File: package.json (hunk 1 above)
Comment (kevkevinpal) [reply package.json:94]: ohh ok gotcha that makes sense, but yea this looks good to me 

just not sure why it only added the `^` and didn't update that minor version number aswell

**Comment by kevkevinpal** [comment]
> > yea this is an issue on github side I can just rerun
> 
> i have seen this multiple times, and it was always bob-lnd that failed when it did, seems like some internal github thing uses that same port (38882)

Yea seems like that might be the issue, its mostly just a minor inconvenience for now

but this change looks good to merge @Evanfeenstra let me know if you've got any comments otherwise I can merge it

**Comment by kevkevinpal** [comment]
@antonilol So I think this PR may have cause the tests to fail as per this issue 

https://github.com/sequelize/sequelize/discussions/14614

So I'm going to try reverting and then seeing if it'll pass the tests

**Comment by antonilol** [comment]
hmmm this PR itself did not fail any tests, and after this got merged i rebased 3 other PRs and they also did not fail any tests, did you update your local npm packages after merging this? did the lock file change after installing, if that is the case packages were modified, you should `git restore package{.json,-lock.json}` and `npm ci`. when packages are installed and the lockfile did not change, you have the exact packages as in the lockfile (the automated tests do), running any tests should produce exact the same results everywhere

**Comment by kevkevinpal** [comment]
> and after this got merged i rebased 3 other PRs and they also did not fail any tests, did you update your local npm packages after merging this

ya I thought the same thing but this was the only thing that was merged within the last few days, not sure if some down stream dependency changed to make typescript more strict which is now causing the tests to fail?

If you want I can try re-running one of your PR's to see if they fail now since nothing's been changed

**Comment by kevkevinpal** [comment]
@antonilol I think I may have figured it out, I think its the version of node/npm that was giving it issues

Check the latests commits here in `.github/workflows/integration_test.yml`
https://github.com/stakwork/sphinx-relay/pull/376

**Comment by kevkevinpal** [comment]
I'll merge this PR after its tests pass and then we can rebase the other PR's

https://github.com/stakwork/sphinx-relay/pull/385

**Comment by antonilol** [comment]
> @antonilol I can revert the revert lmao

without the `^` in the version of sequelize in package.json

**Comment by kevkevinpal** [comment]
> > @antonilol I think I may have figured it out, I think its the version of node/npm that was giving it issues
> > Check the latests commits here in `.github/workflows/integration_test.yml` #376
> 
> did it use v16 before, that didnt work and on node v14 it works?
> 
> i use v16 lts on my sphinx relay

It might be the version of npm and not node version but seems like this sets the npm version aswell

**Pull request merged**